### PR TITLE
[Bug fix]Update teeu_xgboost.md

### PR DESCRIPTION
**Pull request by tarantula-leo**
[Bug fix]：
1. alice&bob is not defined
2. missing bob's use of gen_data()
3. Replace python3.8 with python, the same as teeu.md

# Preamble

For Pull Request initiator, please read the following steps carefully. 请合并请求的发起人仔细阅读下列步骤。

---

If you are creating a `Pull Request` about listed topic below, please go the the `Preview` tab and click the relative link:
如果您要创建一个有关下列内容的的合并请求，请切换到 `Preview` 面板并点击下方相关的链接:

* Select [Awesome-PETs](?expand=1&template=pull_request_template_pets.md) for files under docs/awesome-pets

---

Select a proper `Pull Request` template can accelarate the review process, thanks for your cooperation.
选择一个正确的`合并请求`模版可以加速审查流程，谢谢您的配合

If your `Pull Request` topic is not amoung the list above, please just delete all prepopulated template content.
如果您的`合并请求`内容不在上述的列表里，请删除提前填入的模版内容。

---

Thanks for your contribution to SecretFlow project
感谢您对于 SecretFlow 项目的贡献

### CODE REVIEW

**Comment by tarantula-leo** [comment]
`Python Linter / python-linter / lint `部分问题为项目中其他代码问题，非本次PR改动

**Comment by tarantula-leo** [comment]
`Python Linter / python-linter / lint `部分问题为项目中其他代码问题，非本次PR改动

**Comment by tarantula-leo** [comment]
`Python Linter / python-linter / lint `部分问题为项目中其他代码问题，非本次PR改动

**Comment by tarantula-leo** [comment]
`Python Linter / python-linter / lint `部分问题为项目中其他代码问题，非本次PR改动。

File: docs/tutorial/teeu_xgboost.md [hunk 1]
```diff
@@ -118,11 +118,13 @@ def gen_data():
 # Alice generates its samples.
 x_a, y_a = alice(gen_data, num_returns=2)()
 # Bob generates its samples.
-x_b, y_b = alice(gen_data, num_returns=2)()
+x_b, y_b = bob(gen_data, num_returns=2)()
 
 from secretflow.device import TEEU
 
 # mrenclave can be omitted in simulation mode.
+alice = sf.PYU('alice')
```
Comment (zhouaihui) [inline docs/tutorial/teeu_xgboost.md:126]: alice, bob的定义需要在更前面，在`x_a, y_a = alice(gen_data, num_returns=2)()`之前。

**Comment by tarantula-leo** [comment]
这里的数据是只支持以numpy的形式还是像pandas之类的一样可以？
`res = teeu(xgb_demo)([x_a_teeu, x_b_teeu], [y_a_teeu, y_b_teeu])`

**Review by zhouaihui — APPROVED** [review]
LGTM

**Review by warriorpaw — APPROVED** [review]
LGTM

**Review by ian-huu — APPROVED** [review]
LGTM

**Pull request merged**